src/bitcoincore.rs

```diff
@@ -163,6 +163,20 @@ impl BitcoinRpcApi for BitcoinCoreClient {
         ))
     }
 
+    fn generate_to_address(
+        &self,
+        number_of_blocks: u32,
+        address: &Address,
+    ) -> Result<Result<Vec<BlockHash>, RpcError>, ClientError> {
```

Comment:
:question: Without checking the documentation: The result of `generate_to_address` is probably not a `Vec<BlockHash>`?

Reply:
It is https://bitcoincore.org/en/doc/0.16.0/rpc/generating/generatetoaddress/

Reply:
Interesting! My bad then. We still need a test for the method though :)